### ISSUE TITLE
Strange error in Backend:  Unable to render image tag

### ISSUE THREAD

**Issue by criwe** [sksksksk/skvideo#11]
Hi!

I´ve just update an non composer TYPO3 11 to composer TYPO3 12. Everything fine so far. 
But on pages where skvideo is added an error in the backend occurs:

```
503
Oops, an error occurred!

Unable to render image tag: Folder "/typo3/var/transient/fal-tempfile-12345656.jpg/" does not exist.
```

(the number changes on every page reload).

When removing <f:image src="{imgPath}" alt="" width="200" /> from BePreviewTemplate.html everything works again.

Greets,
Christoph

**Comment by heisenbol** [comment]
hmmmm.... the slash at the end of the filename should not be there. And I would expect a full file path, and not something starting at /typo3 (unless your installation starts at /tpy3 ????

Anyway... I will try to reproduce this and will get back to you. If you have more data e.g. from the typo3 log file, please post it here

**Comment by criwe** [comment]
Hi!
Ok, here is something from the error-log:

 component="TYPO3.CMS.Core.Error.ProductionExceptionHandler": Core: Exception handler (WEB: BE): TYPO3Fluid\Fluid\Core\ViewHelper\Exception, code #1509741911, filetypo3/vendor/typo3/cms-fluid/Classes/ViewHelpers/ImageViewHelper.php, line 221: Unable to render image tag: Folder "typo3/var/transient/fal-tempfile-6081291247064888839.jpg/" does not exist. - {"mode":"WEB","application_mode":"BE","exception_class":"TYPO3Fluid\\Fluid\\Core\\ViewHelper\\Exception","exception_code":1509741911,"file":"typo3/vendor/typo3/cms-fluid/Classes/ViewHelpers/ImageViewHelper.php","line":221,"message":"Unable to render image tag: Folder \"typo3/var/transient/fal-tempfile-6081291247064888839.jpg/\" does not exist.","request_url":"typo3/module/web/layout?token=--AnonymizedToken--&id=21","exception":null}
Thu, 29 Feb 2024 15:48:44 +0100 [CRITICAL] request="e4321bd74a892" component="TYPO3.CMS.Core.Error.ProductionExceptionHandler": Core: Exception handler (WEB: BE): TYPO3Fluid\Fluid\Core\ViewHelper\Exception, code #1509741911, filetypo3/vendor/typo3/cms-fluid/Classes/ViewHelpers/ImageViewHelper.php, line 221: Unable to render image tag: Folder "typo3/var/transient/fal-tempfile-5935274718222510147.jpg/" does not exist. - {"mode":"WEB","application_mode":"BE","exception_class":"TYPO3Fluid\\Fluid\\Core\\ViewHelper\\Exception","exception_code":1509741911,"file":"typo3/vendor/typo3/cms-fluid/Classes/ViewHelpers/ImageViewHelper.php","line":221,"message":"Unable to render image tag: Folder \"typo3/var/transient/fal-tempfile-5935274718222510147.jpg/\" does not exist.","request_url":"typo3/module/web/layout?token=--AnonymizedToken--&id=21&","exception":null}

**Comment by heisenbol** [comment]
ok, I think I got it. The error appears only if you use a custom preview image for the video, and not the image provided by youtube.
I'll try to fix this

**Comment by heisenbol** [comment]
I do not understand why the previous code produces the wrong file path (with the slash at the end). However, a small change fixed the problem in my installation.

Could you please try out the following:
in Classes/Helper.php , line 162, replace
        return $fileRef->getForLocalProcessing();
with
        return $fileRef->getForLocalProcessing(false);

If this works for you, I will upload a newer version

**Comment by criwe** [comment]
Hi!

Yes, that fix works too in my installation. 

Thanks an lot!

Many greetings,
Christoph

**Comment by heisenbol** [comment]
Released in 2.1.1
